### DISCUSSION TITLE
Ok so I missing something.

### DISCUSSION THREAD

**Discussion by bigfranko** [UltimaHoarder/UltimaScraper#1058] [Q&A]
I'm getting the  "Expecting property name enclosed in double quotes: line 14 column 2 (char 435)
Here's my auth section . 
"auth": {
  "username": "default",
  "auth_id": "******",
  "auth_hash": "**********",
  "auth_uniq_": "",
  "sess": "*********",
  "app_token": "********",
  "user_agent": "Mozilla/5.0 (Windows NT 10.0; Win64; x64; rv:88.0) Gecko/20100101 Firefox/88.0",
  "email": "***********",
  "password": "*********",  
  "support_2fa": true,  
  "active": true,
   }

Any help would be greatly appreciated.

**Comment by fightcancer2** [comment]
auth_id, app_token and sess should not be blank.  I think password should be "" instead of "*".

"Make sure your cookies and user-agent are correct. Use this tool Cookie Helper"
https://github.com/DIGITALCRIMINAL/OnlyFans/blob/master/README.md#faq

**Comment by bigfranko** [comment]
Yeah I didn’t want to post my info in  public view,  I put asterisks instead my info.

Sent from Mail<https://go.microsoft.com/fwlink/?LinkId=550986> for Windows 10

**Reply by fightcancer2** [reply]
Cool.  I recommend following the instructions in the FAQ with an emphasis on using "Cookie Helper" to get the most accurate info.
https://github.com/DIGITALCRIMINAL/OnlyFans/blob/master/README.md#faq

**Reply by bigfranko** [reply]
Ok, check this out. I used the cookie helper and now it changed from line 14 column 2  to line 14 column 3.  Am I puting  a { or } in a wrong place

{
    "auth": {
    "auth_hash": "",
    "auth_id": "",
    "auth_uniq_": "",
    "sess": "",
    "app_token": "",
    "user_agent": "Mozilla/5.0 (Windows NT 10.0; Win64; x64; rv:88.0) Gecko/20100101 Firefox/88.0",
    "support_2fa": true,
    "username": "",
    "active": true,
    "email": "",
    "password": "",
  }
 "settings": {

**Reply by fightcancer2** [reply]
Idk.  IMO you should paste everything copied from Cookie Helper ("CH") to your auth.json file.

So click Copy to Clipboard in CH.
Go to your auth.json file.
Select all there (Ctrl + A).
Paste.
Close and save your auth.json file.

If that doesn't work, reinstall with the newest update that came out less than 48 hours ago (which will involve doing the steps above again in order to update your auth.json file).

**Reply by bigfranko** [reply]
Reading is fundamental. LOL I figured out what I was doing wrong. I got 646 to work.  Thanks for the help.

**Reply by LunarPenguin** [reply]
If your problem is solved please mark the discussion as answered
